src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="post" class="form"
+    onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <lable class="field" for="user_surname">
```

Comment:
```suggestion
        <label class="field" for="user_surname">
```
Actually, this tag is called `label` (not `lable`)